### PR TITLE
Update README.md

### LINKED ISSUE #600: [FEATURE] Links containing the same content should be tagged as Alternatives
### Description

Links that contain the same content should be tagged as "Alternatives" (or something similar) to avoid confusion.

### Screenshots

image

### Additional information

I can help with this change.

### PR DESCRIPTION
**Pull request by KUNDAN1334**
i just numbered the links so that confusion can be reduced and add alt resource to same content.

## Changes proposed
I just numbered the link and content so that confusion can be reduced and also write alt resource to same content.

1. numbered the content
2. provide alt resource tag

### CODE REVIEW

**Review by FlashBang-000X — APPROVED** [review]

**Comment by KUNDAN1334** [comment]
after reviewed why this showing review required

File: Frontend-Development/README.md [hunk 1]
```diff
@@ -15,14 +15,14 @@
...
-- [How Does the Internet Work? - Lesics(Youtube)](https://www.youtube.com/watch?v=x3c1ih2NJEg)
-- [What is DNS? | How DNS works](https://www.cloudflare.com/en-gb/learning/dns/what-is-dns/)
+- [1] [Computer Networking course - Kunal Khushwaha(Youtube)](https://www.youtube.com/watch?v=IPvYjXCsTg8)
+- [2] [How the Internet Travel Across Oceans? - TechVision(Youtube)](https://www.youtube.com/watch?v=yd1JhZzoS6A)
+- [3] [How does the Internet Work? - Vox(Youtube)](https://www.youtube.com/watch?v=TNQsmPf24go)
+- [4] [How does the Internet Work? - Mozilla web docs](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/How_does_the_Internet_work)
+- [5] [How Does the Internet Work? - Stanford.edu](https://web.stanford.edu/class/msande91si/www-spr04/readings/week1/InternetWhitepaper.htm)
+- [6] [How Does the Internet Work? - cloudflare](https://www.cloudflare.com/en-in/learning/network-layer/how-does-the-internet-work/)
+- [7] [How Does the Internet Work? - Lesics(Youtube)](https://www.youtube.com/watch?v=x3c1ih2NJEg)
+- [8] [What is DNS? | How DNS works](https://www.cloudflare.com/en-gb/learning/dns/what-is-dns/)
```
Comment (siddhant-khisty) [inline Frontend-Development/README.md:25]: These numbers are not required. Rather the way you have mentioned Alt resources in the CSS section seems like a much better way to go about it

File: Frontend-Development/README.md [hunk 2]
```diff
@@ -53,19 +51,19 @@
...
+- [4] ALT RESOURCE---> [CSS zero to hero - FreeCodeCamp(Youtube)](https://www.youtube.com/watch?v=1Rs2ND1ryYc)
+- [5] [Kevin Powell(Youtube)](https://www.youtube.com/kepowob)
+- [6] [Introduction to CSS - Web Dev Simplified](https://www.youtube.com/playlist?list=PLZlA0Gpn_vH9D0J0Mtp6lIiD_8046k3si)
+- [7] [The guide to responsive web design](https://webflow.com/blog/responsive-web-design)
+- [8] [Learn flexbox by gaming](https://flexboxfroggy.com/)
+- [9] ALT RESOURCE---> [A complete guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)
+- [10] [Learn Grid by gaming](https://cssgridgarden.com/)
+- [11] ALT RESOURCE---> [A complete guide to CSS Grid]( https://css-tricks.com/snippets/css/complete-guide-grid/)
+- [12] [Learn Hex Code by gaming](http://www.hexinvaders.com/)
+- [13] [CSS Diner](https://flukeout.github.io/)
```
Comment (siddhant-khisty) [inline Frontend-Development/README.md:66]: Please remove the numbers. Only keep the `ALT RESOURCE` part

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]

**Comment by siddhant-khisty** [comment]
Hey @KUNDAN1334 There have been some updates with regard to this issue. Please have a look over at #600 for the same

**Review by AmanKumarVerma11 — APPROVED** [review]

**Pull request closed without merging**